src/components/events/EventsSection.jsx

```diff
@@ -22,11 +22,20 @@ class EventsSection extends React.Component {
     getEvents().then((response) => {
       let events = Object.values(response.data).map(event => {
         return humps.camelizeKeys(event);
-      });
+	  });
+	  console.log(events);
       this.setState({ events, loading: false });
    })
   }
 
+  handleUpdate = (event) => {
+	
+	let events = [...this.state.events.filter(e => e.id !== event.id)];
+	events.push(event);
+	console.log(events);
+	this.setState({ events });
```

Comment:
lets put humps here so that we can pass anything in and ensure itll be camelCased